doc/contributing: info about where the docs are located (#3728)
replicatedhq/kURL: CONTRIBUTING.md

### QEMU on MacOS

You might want try to use [UTM](https://mac.getutm.app/).

## Documentation

If you would like to contribute with the docs published in https://kurl.sh/docs/introduction/ then,

> see that they are built from the repository [replicatedhq/kurl.sh][kulr-docs].

Reviewer: do you have this target already? `kulr-docs` also it is misspelled

Author: Sorry, it came from the split. 
I added all links with alias but in the review was requested not to use alias. 
Fixing that now.

You might want try to use [UTM](https://mac.getutm.app/).

## Documentation

If you would like to contribute with the docs published in https://kurl.sh/docs/introduction/ then,

> see that they are built from the repository [replicatedhq/kurl.sh][kulr-docs].

Author: ```suggestion
see that they are built from the repository [replicatedhq/kurl.sh](https://github.com/replicatedhq/kurl.sh).
```

You might want try to use [UTM](https://mac.getutm.app/).

## Documentation

> If you would like to contribute with the docs published in https://kurl.sh/docs/introduction/ then,
> see that they are built from the repository [replicatedhq/kurl.sh](https://github.com/replicatedhq/kurl.sh).

Reviewer: ```suggestion

For contributions to the kURL documentation published at https://kurl.sh/docs/introduction/,
see the source repository at [replicatedhq/kurl.sh](https://github.com/replicatedhq/kurl.sh).
```
